stdlib/experimental/unpivot.go

```diff
@@ -150,12 +148,17 @@ func (t *unpivotTransformation) Process(chunk table.Chunk, d *execute.TransportD
 			buffer.Values[toColumn] = values
 		}
 
-		buffer.Values[len(buffer.Values)-3] = array.StringRepeat(c.Label, newChunkLen, mem)
+		oldValues := chunk.Values(i)
+
+		if timeColumn != -1 {
+			// The time array does not contain any nulls so we use the value array for that information
+			times := array.CopyValidValues(mem, chunk.Values(timeColumn), oldValues)
+			buffer.Values[len(buffer.Values)-3] = times
```

Comment:
I wonder if this fix here will address the bug that @scbrickley found with the new `testing.diff`... and this also explains why it seemed to be specific to IOx